traffic_ops/traffic_ops_golang/api/shared_interfaces.go

```diff
@@ -58,6 +58,10 @@ type Creator interface {
 	Validator
 }
 
+type AllowMultipleCreates interface {
```

Comment:
Please add a comment for the `AllowMultipleCreates` interface.

Reply:
done